README.md

```diff
@@ -11,10 +11,9 @@ The 2 sections I added to build for this project are:
 
 ## Live Demo (if available)
 github repo link 
-[https://github.com/ManarOmari/Project-1-Setup-and-mobile-version-skeleton/pull/1]
+[https://github.com/ManarOmari/Project-1-Setup-and-mobile-version-skeleton/]
 
-online version link 
-none 
+online version link [https://manaromari.github.io/Project-1-Setup-and-mobile-version-skeleton/]
```

Comment:
[Optional] Kindly use the proper markdown link format to include the link. This will help to make it that the link url (which might be long) doesn't get displayed, only the text you want will show up.

A display as this will be great:
![image](https://user-images.githubusercontent.com/20554583/187189787-294e32a9-fbf1-444b-895d-cf106979517c.png)